### ISSUE TITLE
Ignored API operation, but not ignored object definition

### ISSUE THREAD

**Issue by patvong** [stoicflame/enunciate#377]
I have an endpoint that I ignore:
```
	/**
	 * Add docs
	 *
	 * @param applicationId Application id
	 * @param name  Name
	 * @return Validation object
	 */
	@GET
	@Path("/abc")
	@Consumes({ MediaType.APPLICATION_JSON })
	@Produces({ MediaType.APPLICATION_JSON })
	@TypeHint(ValidationWS.class)
	@Facet("Ignore")
	@Ignore
	Response validateTransferability(@PathParam("applicationId") Long applicationId, @QueryParam("name") String name);
```
I would have expected to not be included the in the swagger.json file the object definition of `ValidationWS`, but it is included even though I added @Ignore and @Facet("Ignore") for that class.

To make it ignore correctly, I need to remove `@TypeHint(ValidationWS.class)`. Is it the expected behavior?

**Comment by stoicflame** [comment]
Probably a bug. Thanks for the report.

**Comment by stoicflame** [comment]
Looked into this. The `@Facet` mechanism is applied as a post-filter because different modules might have different facets enabled/disabled. So at the time that the type definitions are being added to the model, you can't consult the facets because some modules might have the elements excluded, while others might have them included.

However, the `@Ignore` annotation should have removed that method and any associated references to it, so I fixed that at 6a9602c.

**Comment by stoicflame** [comment]
[Enunciate 2.4 has been released](https://github.com/stoicflame/enunciate/releases/tag/v2.4.0).
